core/sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/Expressions.java

```diff
@@ -318,6 +325,14 @@ public static Expression<?> notEquals(Operand left, Operand right) {
 		return binaryExpression(BinaryOperator.NOT_EQUALS, left, right);
 	}
 
+	public static Expression<?> notEquals(Variable left, RdfValue right) {
+		return binaryExpression(BinaryOperator.NOT_EQUALS, left, right);
+	}
+
+	public static Expression<?> notEquals(Variable left, IRI right) {
+		return binaryExpression(BinaryOperator.NOT_EQUALS, left, iri(right));
+	}
+
```

Comment:
Are these changes required for the fix in TransactionObject?

Reply:
As long as the issues in the 4.1.0-M1 version are present, we desperately need these changes already in 4.0.4 for our project.

Reply:
@maxstolze but these specific changes have nothing to do with the fix that you need. They're just a couple of utility functions you added.